components/cli/vendor/github.com/docker/docker/pkg/archive/archive_linux.go

```diff
@@ -5,9 +5,9 @@ import (
 	"os"
 	"path/filepath"
 	"strings"
-	"syscall"
 
 	"github.com/docker/docker/pkg/system"
+	"golang.org/x/sys/unix"
```

Comment:
Brought in with this PR:
* https://github.com/moby/moby/pull/33399 [project] Switch most syscalls to golang.org/x/sys